Nulecule.yaml

```diff
@@ -0,0 +1,95 @@
+# Put all metadata here (no more separate id, specversion objects, etc.
+metadata:
+  name: Hello World
+  id: helloworld
+  appversion: 0.0.1
+  spec: 1.0.0
```

Comment:
Hi @cdrage, do we need to write the spec version in each Nulecule file? In fact, most of the things in metadata.
Can it be something like, we set all the sensible default values for each field, such that if the user specifies the fields in his/her Nulecule, those will be overridden, but even if he does not, it's fine, the defaults are already set!